projects/driver_controls/inc/event_arbiter.h

```diff
@@ -13,15 +13,25 @@
 // Arbitrary FSM cap
 #define EVENT_ARBITER_MAX_FSMS 10
 
+typedef struct EventArbiterOutputData {
+  uint8_t id;
+  uint8_t state;
+  uint16_t data;
+} EventArbiterOutputData;
+
 // Typedef for the FSM arbitration functions
 typedef bool (*EventArbiterCheck)(const Event *e);
 
-// Initializes the event arbiter to the default state
-StatusCode event_arbiter_init(void);
+typedef void (*EventArbiterOutput)(FSM *fsm, EventArbiterOutputData data);
```

Comment:
Why does this take an FSM? I see no reason why the output function should be even be aware that FSMs exist.

Also, if it really needed it it should be const.